Justfile

```diff
@@ -9,6 +9,8 @@ GOBIN := env("GOBIN", `go env GOPATH`+"/bin")
 DIST_FOLDER := if GOARCH == "amd64" { "dist/vcluster_linux_amd64_v1" } else if GOARCH == "arm64" { "dist/vcluster_linux_arm64_v8.0" } else { "unknown" }
```

Comment:
All the places you'll need to change the filename and docs to match this. [This search lists 4](https://github.com/search?q=repo%3Aloft-sh%2Fvcluster%20vcluster-images.txt&type=code), but I also see one in contributing.md.

Reply:
Added that